core/domain/collection_domain_test.py

```diff
@@ -1077,3 +1079,38 @@ def test_validation_fails_with_invalid_contributor_id_in_contributor_ids(
             utils.ValidationError,
             'Expected each id in contributor_ids to be string, received 2'):
             self.collection_summary.validate()
+
+    def test_is_private(self):
+        self.assertTrue(self.collection_summary.is_private())
+        self.collection_summary.status = constants.ACTIVITY_STATUS_PUBLIC
```

Comment:
I feel this is something a setter should do, so we shouldn't be touching the raw stuff out here.

Reply:
I would agree with you in regular files but I think that we don!t need to have a setter just for tests. If you think we should get rid of this I think it would be better to replace it with a newly constructed domain object.

Reply:
Yeah, let's have a newly constructed domain object for these.